kubectl-pg/cmd/addDb.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"log"
+)
+
+// addDbCmd represents the addDb command
+var addDbCmd = &cobra.Command{
+	Use:   "add-db",
+	Short: "Adds a DB and its owner to a Postgres cluster. The owner role is created if it does not already exist",
```

Comment:
change this to have a clear separation between `Short` and `Long`  versions of help. For the short version, a single sentence is enough. The long version should mention corner cases and restrictions.